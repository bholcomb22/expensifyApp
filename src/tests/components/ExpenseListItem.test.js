import React from 'react'
import {shallow} from 'enzyme'
import ExpenseListItem from '../../components/ExpenseListItem'
import expenses from '../fixtures/expenses'

const testExpense = expenses[1]

test('should render expense list item correctly', () => {
    const wrapper = shallow(<ExpenseListItem {...testExpense} />)
    expect(wrapper).toMatchSnapshot()
})