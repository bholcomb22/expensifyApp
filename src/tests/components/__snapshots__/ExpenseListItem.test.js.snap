// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense list item correctly 1`] = `
<div>
  <Link
    to="/edit/2"
  >
    <h3>
      Description:
      Rent
    </h3>
  </Link>
  <p>
    $100.00
    -
    December 27th, 1969
  </p>
</div>
`;
